### DISCUSSION TITLE
CrossProbingApp issue

### DISCUSSION THREAD

**Discussion by paprophet** [iTwin/itwinjs-core#500] [Q&A] [answered]
now.i want use CrossProbingApp from sample,but i have two issue:
1.how i can change the revit file (*.rvt) to the bim file by  use this sample . (by iTwin Snapshot.exe app)
2.in the UiFramework,how i can split window?

**Comment by calebmshafer** [comment]
@roopksaini can you try and answer @paprophet 's question about the CrossProbing sample?

**Comment by roopksaini** [comment, marked as answer]
Hello @paprophet,

1) iTwin Snapshot works similar to the generate snapshot functionality in the iTwin Synchronizer. You can watch a demo right [here](https://youtu.be/fYcQai6x3PA?t=1514).
2) In the cross-probing sample, I am using basic CSS to display the two viewports (code for [viewport 1](https://github.com/imodeljs/frontend-sample-showcase/blob/master/src/frontend-samples/cross-probing-sample/CrossProbingUI.tsx#L58-L59) and [viewport 2](https://github.com/imodeljs/frontend-sample-showcase/blob/master/src/frontend-samples/cross-probing-sample/CrossProbingUI.tsx#L49-L50)). Then it's just a matter of fetching either viewport depending on which type of model it displays ([2D or 3D](https://github.com/imodeljs/frontend-sample-showcase/blob/master/src/frontend-samples/cross-probing-sample/CrossProbingApp.tsx#L80-L94)).
